source/blog/2017-11-05-vdsm-setup-networks-is-now-much-quicker.html.md

```diff
@@ -0,0 +1,40 @@
+---
+title: VDSM setupNetworks is now much quicker
+author: phoracek
+tags: ovirt, 4.2.0, network
+date: 2017-11-05 09:00:00 CET
+comments: true
+published: true
+---
```

Comment:
please start with something like
"""
Assume you have an oVirt cluster with hundreds of VM networks. Now you add a new host to the cluster. In order for it to move to the `Operational` state, it must have all required networks attached to it.
"""

Reply:
Done